code/_onclick/hud/alert.dm

```diff
@@ -1034,39 +1018,34 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 
 // PRIVATE = only edit, use, or override these if you're editing the system as a whole
 
+/// Gets the placement for the alert based on its index
+/datum/hud/proc/get_ui_alert_placement(index)
+	// Only has support for 5 slots currently
+	if(index > 5)
+		return ""
+
+	return "EAST-1:28,CENTER+[6 - index]:[29 - index * 2]"
```

Comment:
```suggestion
	return "EAST-1:28,CENTER+[6 - index]:[29 - (index * 2)]"
```

lest the PEMDAS monster eat our liver for i live in great fear of that happening